src/lib/components/AppBar.svelte

```diff
@@ -78,6 +78,18 @@
       <li>
         <a id="blog" href="/blog">blog</a>
       </li>
+      <li>
+        <a
+          id="shop"
+          href="https://the-blend-shop.square.site/"
+          target="_blank"
+          rel="noopener noreferrer"
+          on:click={() => window.gtag('event', 'click_shop')}
```

Comment:
I also added a google analytics event here, will need to configure it there i think